src/App.js

```diff
@@ -1,14 +1,12 @@
 import React from 'react';
-import './App.css';
+import BooksList from './containers/BooksList';
+import BooksForm from './components/BooksForm';
```

Comment:
this is normally fine, but for the sake of the requirements of this project, please relocate this file into the `src/components/` folder